tds/settings.py

```diff
@@ -19,7 +19,7 @@ class Settings(BaseSettings):
     DKG_URL = "http://34.230.33.149"
     DKG_API_PORT = 8771
     DKG_DESC_PORT = 8772
-    NEO4J_ENABLED = True
+    NEO4J_ENABLED = False
```

Comment:
Do we want to this to be False by default? `NEO4J_ENABLED` could be set to `True` in `api.env.sample` so we wouldn't have to change it here.